src/index.ts

```diff
@@ -408,20 +431,18 @@ export default class Dalai {
     cwd?: string,
     cb?: (payload: string) => void
   ) {
-    const escapedArgs = args
-      .map((x) => `"${this.escapeShellArg(x)}"`)
-      .join(" ");
-
     return new Promise((resolve, reject) => {
       const config: pty.IPtyForkOptions | pty.IWindowsPtyForkOptions =
         Object.assign({}, this.config);
+
       if (cwd) {
         config.cwd = path.resolve(cwd);
       }
-      console.log(
-        `exec: ${cmd}, with args: ${escapedArgs} in working dir: ${config.cwd}`
-      );
-      const ptyProcess = pty.spawn(shell, [], config);
+
+      const commandToWrite = `${cmd} ${args.join(" ")}\r`;
+      console.log(`🤖 writing command: ${commandToWrite}`);
```

Comment:
That console log and argument could be changed to something more accurate. It's no longer writing to shell's stdin.